src/webapp/components/app/App.tsx

```diff
@@ -14,8 +12,11 @@ import { Router } from "../../pages/Router";
 import { useMigrations } from "../migrations/hooks";
 import Migrations from "../migrations/Migrations";
 import Share from "../share/Share";
+import { Feedback } from "@eyeseetea/feedback-component";
 import "./App.css";
 import { muiTheme } from "./themes/dhis2.theme";
+import {Maybe} from "../../../types/utils";
+import {AppConfig} from "./AppConfig";
```

Comment:
Normally, prettier would edit it as `{ AppConfig }`. Optionally you can also run `yarn prettify` and it will do the work for you; but we normally have the setting "Format on save" with Prettier.